ontopy/excelparser.py

```diff
@@ -555,3 +433,221 @@ def _add_literal(  # pylint: disable=too-many-arguments
                 warnings.warn(f"Missing metadata {name}")
             else:
                 warnings.warn(f"{data[0]} has no {name}")
+
+
+def _clean_dataframe(
+    data: pd.DataFrame,
+) -> pd.DataFrame:
+    """Remove lines with empty prefLabel,
+    convert all data to strings, remove spaces, and finally remove
+    additional rows with 0-length prefLabel.
+    """
+    data = data[data["prefLabel"].notna()]
+    data = data.astype(str)
+    data["prefLabel"] = data["prefLabel"].str.strip()
+    data = data[data["prefLabel"].str.len() > 0]
+    data.reset_index(drop=True, inplace=True)
+    return data
+
+
+def _add_concepts(
```

Comment:
Just a minor comment: In OWL are concepts called classes, so I would have called it _add_class().
Don't feel any urge to change...

Reply:
closed with #620